src/app/pages/onboarding/set-password/set-password.tsx

```diff
@@ -105,7 +105,10 @@ export const SetPasswordPage = () => {
       }),
     confirmPassword: yup
       .string()
-      .defined()
+      .when('password', {
+        is: (val: string | any[]) => (val && val.length > 0 ? true : false),
```

Comment:
```suggestion
        is: (val: string | any[]) => (val && val.length > 0),
```

`val && val.length > 0` alone is an expression returning true or false, so the ternary is unnecessary

Reply:
Good point, but it was here already ...I will change it though.